src/main/resources/application-test.yml

```diff
@@ -0,0 +1,15 @@
+application:
```

Comment:
it should be in `it` directory, not `main`

Reply:
moved to the it directory